ducc/src/tests/ducc.rs

```diff
@@ -31,6 +31,16 @@ fn no_duktape_global() {
     assert!(!globals.contains_key("Duktape").unwrap());
 }
 
+#[test]
+fn inspect_callstack() {
+    let ducc = Ducc::new();
+    ducc.globals().set("fun", ducc.create_function(|inv| {
+        inv.ducc.inspect_callstack_entry(-1).expect("callstack entry was None");
+        Ok(())
+    })).unwrap();
+    ducc.globals().get::<_, Function>("fun").unwrap().call::<(), ()>(()).unwrap();
+}
+
```

Comment:
Just add `use function::Function;` to this file and I'll merge! (it doesn't compile currently)

Reply:
Juggling with Github editor has its downsides :d

Reply:
Indeed... with your latest change you don't need to `use function::Function` at all ;)